docs/PLUGIN_CATALOG.md

```diff
@@ -122,12 +124,8 @@ There will always be more plugins we wish we had. To make sure others can contri
 
 | Issue | Type | Description |
 |-------|------|-------------|
-| [#1612](https://github.com/intelsdi-x/snap/issues/1612) | processor | Plugin wanted: threshold processor |
-| [#1611](https://github.com/intelsdi-x/snap/issues/1611) | processor | Plugin wanted: change detector processor |
+| [#1635](https://github.com/intelsdi-x/snap/issues/1635) | collector | Plugin wanted: pidstat collector |
 | [#1607](https://github.com/intelsdi-x/snap/issues/1607) | collector | Collector for Postgresql? |
-| [#1567](https://github.com/intelsdi-x/snap/issues/1567) | collector | Plugin wanted: InfluxDB data collector |
-| [#1566](https://github.com/intelsdi-x/snap/issues/1566) | processor | Plugin wanted: logs filter |
-| [#1565](https://github.com/intelsdi-x/snap/issues/1565) | publisher | Plugin wanted: Slack publisher |
 | [#1447](https://github.com/intelsdi-x/snap/issues/1447) | collector | Snap collector |
 | [#1436](https://github.com/intelsdi-x/snap/issues/1436) | collector | RDT metrics collector |
```

Comment:
RDT collector is available and may land on plugin catalog - see https://github.com/intelsdi-x/snap/issues/1436